exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/FragmentWrapper.java

```diff
@@ -49,58 +51,135 @@ public String getId() {
+    tb.appendCell(complete.size() + " / " + major.getMinorFragmentProfileCount(), null, null);
 
     // If there are no stats to aggregate, create an empty row
     if (complete.size() < 1) {
-      tb.appendRepeated("", null, NUM_NULLABLE_OVERVIEW_COLUMNS);
+      tb.appendRepeated("", null, NUM_NULLABLE_ACTIVE_OVERVIEW_COLUMNS, null);
       return;
     }
 
     final MinorFragmentProfile firstStart = Collections.min(complete, Comparators.startTime);
     final MinorFragmentProfile lastStart = Collections.max(complete, Comparators.startTime);
-    tb.appendMillis(firstStart.getStartTime() - start, null);
-    tb.appendMillis(lastStart.getStartTime() - start, null);
+    tb.appendMillis(firstStart.getStartTime() - start, null, null);
+    tb.appendMillis(lastStart.getStartTime() - start, null, null);
 
     final MinorFragmentProfile firstEnd = Collections.min(complete, Comparators.endTime);
     final MinorFragmentProfile lastEnd = Collections.max(complete, Comparators.endTime);
-    tb.appendMillis(firstEnd.getEndTime() - start, null);
-    tb.appendMillis(lastEnd.getEndTime() - start, null);
+    tb.appendMillis(firstEnd.getEndTime() - start, null, null);
+    tb.appendMillis(lastEnd.getEndTime() - start, null, null);
 
-    long total = 0;
+    long totalDuration = 0L;
+    double totalProcessInMillis = 0.0d;
+    double totalWaitInMillis = 0.0d;
     for (final MinorFragmentProfile p : complete) {
-      total += p.getEndTime() - p.getStartTime();
+      totalDuration += p.getEndTime() - p.getStartTime();
+      //Capture Busy & Wait Time
+      List<OperatorProfile> opProfileList = p.getOperatorProfileList();
+      for (OperatorProfile operatorProfile : opProfileList) {
+        totalProcessInMillis += operatorProfile.getProcessNanos()/1E6;
+        totalWaitInMillis += operatorProfile.getWaitNanos()/1E6;
+      }
     }
 
     final MinorFragmentProfile shortRun = Collections.min(complete, Comparators.runTime);
     final MinorFragmentProfile longRun = Collections.max(complete, Comparators.runTime);
-    tb.appendMillis(shortRun.getEndTime() - shortRun.getStartTime(), null);
-    tb.appendMillis(total / complete.size(), null);
-    tb.appendMillis(longRun.getEndTime() - longRun.getStartTime(), null);
+    tb.appendMillis(shortRun.getEndTime() - shortRun.getStartTime(), null, null);
+    tb.appendMillis(totalDuration / complete.size(), null, null);
+    tb.appendMillis(longRun.getEndTime() - longRun.getStartTime(), null, null);
+
+    tb.appendPercent(totalProcessInMillis / (totalProcessInMillis + totalWaitInMillis), null,
+      String.format("&#8721;Busy: %,.2fs + &#8721;Wait: %,.2fs", totalProcessInMillis/1E3, totalWaitInMillis/1E3));
 
     final MinorFragmentProfile lastUpdate = Collections.max(complete, Comparators.lastUpdate);
-    tb.appendTime(lastUpdate.getLastUpdate(), null);
+    tb.appendMillis(System.currentTimeMillis()-lastUpdate.getLastUpdate(), null, new SimpleDateFormat().format(new Date(lastUpdate.getLastUpdate())));
```

Comment:
This is actually a tooltip, to indicate the time on the clock when the last update/progress was made. Since we already have a delta, this might not be needed. The only time I can imagine is when I'd want to refer to that to go through the logs to narrow down on the cause of a slow-down/hang in a query. 
Do you think that would help, or should I just delete the tooltip, @paul-rogers ?